templates/layouts/layout-both-columns.tpl

```diff
@@ -112,5 +112,7 @@
     {block name='hook_before_body_closing_tag'}
       {hook h='displayBeforeBodyClosingTag'}
     {/block}
+    
+    {include file='components/toast-container.tpl'}
```

Comment:
Prefer adding it to bottom_elements to keep consistency, `page-loader` is an utility component like this one